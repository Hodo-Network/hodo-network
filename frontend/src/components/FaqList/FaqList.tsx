import ReactHtmlParser from "react-html-parser";
import { FAQ } from "../../typings/faq";

export interface FaqListProps {
  items: Array<FAQ>;
}

export const FaqList = ({ items }: FaqListProps) => {
  return (
    <dl className='space-y-8 divide-y divide-base-300'>
      {items.map((item) => (
        <div key={item.id} className='pt-6 md:grid md:grid-cols-12 md:gap-8'>
          <dt className='text-base font-medium md:col-span-5'>
            {item.question}
          </dt>
          <dd className='mt-2 md:mt-0 md:col-span-7 prose-primary'>
            <p>
              {ReactHtmlParser(item.answer)}
            </p>
          </dd>
        </div>
      ))}
    </dl>
  );
};
