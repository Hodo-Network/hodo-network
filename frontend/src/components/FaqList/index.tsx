import { FaqList } from "./FaqList";
export default FaqList;
