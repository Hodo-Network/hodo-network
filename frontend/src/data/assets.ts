import { NFT } from '../typings/nft';
// https://www.latlong.net/
// https://gateway.pinata.cloud/ipfs/QmWdDjeh7R5K9w7uzh7bSMgmuDoNTMYXVY5ER69LdHwWc4

export const assets: Array<NFT> = [
    {
        id: '1000',
        tokenId: '0',
        tokenURI: '',
        name: "Cape Town",
        description: 'Capital City of South Africa.',
        image: "https://images.unsplash.com/photo-1452838269443-c86e6a3134f9?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=512&q=80",
        lat: -33.924870,
        lng: 18.424055,
        contractAddress: '0x0000000000000000000000000000000000000000',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        creator: '0x000',
        supply: '1',
        amount: '100',
        price: 2,
        created_at: '1634233032',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1001',
        tokenId: '1',
        tokenURI: '',
        name: "Bangkok",
        description: 'Capital City of Thailand.',
        image: "https://images.unsplash.com/photo-1452838269443-c86e6a3134f9?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=512&q=80",
        lat: 13.756331,
        lng: 100.501762,
        contractAddress: '0x0000000000000000000000000000000000000000',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1002',
        tokenId: '2',
        tokenURI: '',
        name: "Jakarta",
        description: 'Capital City of Indonesia.',
        image: "https://images.unsplash.com/photo-1452838269443-c86e6a3134f9?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=512&q=80",
        lat: -6.175110,
        lng: 106.865036,
        contractAddress: '0x0000000000000000000000000000000000000000',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1003',
        tokenId: '3',
        tokenURI: '',
        name: "Guatemala City",
        description: 'Capital City of Guatemala.',
        image: "https://images.unsplash.com/photo-1452838269443-c86e6a3134f9?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=512&q=80",
        lat: 14.634915,
        lng: -90.506882,
        contractAddress: '0x0000000000000000000000000000000000000000',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1004',
        tokenId: '4',
        tokenURI: '',
        name: "Jamestown",
        description: 'Capital City of Saint Helena.',
        image: "https://images.unsplash.com/photo-1452838269443-c86e6a3134f9?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=512&q=80",
        lat: 36.434269,
        lng: -84.935371,
        contractAddress: '0x0000000000000000000000000000000000000000',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1005',
        tokenId: '5',
        tokenURI: '',
        name: "Afghanistan",
        description: 'Here is an NFT description 1.',
        image: "https://images.unsplash.com/photo-1574158676804-4d719c7eb7c0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=512&q=80",
        lat: 33.93911,
        lng: 67.709953,
        contractAddress: '0x0000000000000000000000000000000000000000',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1006',
        tokenId: '6',
        tokenURI: '',
        name: "Afghanistan",
        description: 'Here is an NFT description 2.',
        image: "https://images.unsplash.com/photo-1574158676804-4d719c7eb7c0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=512&q=80",
        lat: 33.93911,
        lng: 67.709953,
        contractAddress: '0x0000000000000000000000000000000000000000',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1007',
        tokenId: '7',
        tokenURI: '',
        name: "Afghanistan",
        description: 'Here is an NFT description 3.',
        image: "https://images.unsplash.com/photo-1574158676804-4d719c7eb7c0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=512&q=80",
        lat: 33.93911,
        lng: 67.709953,
        contractAddress: '0x0000000000000000000000000000000000000000',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1008',
        tokenId: '8',
        tokenURI: '',
        name: "Andorra",
        description: 'Here is an NFT description 4.',
        image: "https://images.unsplash.com/photo-1574158676804-4d719c7eb7c0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=512&q=80",
        lat: 42.506285,
        lng: 1.521801,
        contractAddress: '0x0000000000000000000000000000000000000000',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1009',
        tokenId: '9',
        tokenURI: '',
        name: "Australia",
        description: 'Here is an NFT description 5.',
        image: "https://images.unsplash.com/photo-1574158676804-4d719c7eb7c0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=512&q=80",
        lat: -25.274398,
        lng: 133.775136,
        contractAddress: '0x0000000000000000000000000000000000000000',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1010',
        tokenId: '10',
        tokenURI: '',
        name: "Argentina",
        description: 'Here is an NFT description 6.',
        image: "https://images.unsplash.com/photo-1574158676804-4d719c7eb7c0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=512&q=80",
        lat: -38.416097,
        lng: -63.616672,
        contractAddress: '0x0000000000000000000000000000000000000000',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
];