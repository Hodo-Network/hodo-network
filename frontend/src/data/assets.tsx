import { NFT } from '../typings/nft';
// https://www.latlong.net/
// https://gateway.pinata.cloud/ipfs/QmWdDjeh7R5K9w7uzh7bSMgmuDoNTMYXVY5ER69LdHwWc4

export const assets: Array<NFT> = [
    {
        id: '1000',
        tokenId: '0',
        tokenURI: '',
        name: "Cape Town",
        description: 'Capital City of South Africa.',
        image: "/assets/images/skyline.jpeg",
        lat: -33.924870,
        lng: 18.424055,
        contractAddress: '0x0000000000000000000000000000000000000000',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        creator: '0x000',
        supply: '1',
        amount: '100',
        price: 2,
        created_at: '1634233032',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1001',
        tokenId: '1',
        tokenURI: '',
        name: "Bangkok",
        description: 'Capital City of Thailand.',
        image: "/assets/images/skyline.jpeg",
        lat: 13.756331,
        lng: 100.501762,
        contractAddress: '0x0000000000000000000000000000000000000002',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1002',
        tokenId: '2',
        tokenURI: '',
        name: "Jakarta",
        description: 'Capital City of Indonesia.',
        image: "/assets/images/skyline.jpeg",
        lat: -6.175110,
        lng: 106.865036,
        contractAddress: '0x0000000000000000000000000000000000000002',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1003',
        tokenId: '3',
        tokenURI: '',
        name: "Guatemala City",
        description: 'Capital City of Guatemala.',
        image: "/assets/images/skyline.jpeg",
        lat: 14.634915,
        lng: -90.506882,
        contractAddress: '0x0000000000000000000000000000000000000002',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1004',
        tokenId: '4',
        tokenURI: '',
        name: "Jamestown",
        description: 'Capital City of Saint Helena.',
        image: "/assets/images/skyline.jpeg",
        lat: 36.434269,
        lng: -84.935371,
        contractAddress: '0x0000000000000000000000000000000000000002',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1005',
        tokenId: '5',
        tokenURI: '',
        name: "Afghanistan",
        description: 'Here is an NFT description 1.',
        image: "/assets/images/forest.jpeg",
        lat: 33.93911,
        lng: 67.709953,
        contractAddress: '0x0000000000000000000000000000000000000001',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1006',
        tokenId: '6',
        tokenURI: '',
        name: "Afghanistan",
        description: 'Here is an NFT description 2.',
        image: "/assets/images/forest.jpeg",
        lat: 33.93911,
        lng: 67.709953,
        contractAddress: '0x0000000000000000000000000000000000000001',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1007',
        tokenId: '7',
        tokenURI: '',
        name: "Afghanistan",
        description: 'Here is an NFT description 3.',
        image: "/assets/images/forest.jpeg",
        lat: 33.93911,
        lng: 67.709953,
        contractAddress: '0x0000000000000000000000000000000000000001',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1008',
        tokenId: '8',
        tokenURI: '',
        name: "Andorra",
        description: 'Here is an NFT description 4.',
        image: "/assets/images/forest.jpeg",
        lat: 42.506285,
        lng: 1.521801,
        contractAddress: '0x0000000000000000000000000000000000000001',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1009',
        tokenId: '9',
        tokenURI: '',
        name: "Australia",
        description: 'Here is an NFT description 5.',
        image: "/assets/images/forest.jpeg",
        lat: -25.274398,
        lng: 133.775136,
        contractAddress: '0x0000000000000000000000000000000000000001',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
    {
        id: '1010',
        tokenId: '10',
        tokenURI: '',
        name: "Argentina",
        description: 'Here is an NFT description 6.',
        image: "/assets/images/forest.jpeg",
        lat: -38.416097,
        lng: -63.616672,
        contractAddress: '0x0000000000000000000000000000000000000001',
        contractName: 'Contract name',
        contractVerified: 0,
        owner_address: '0x',
        created_at: '1634233032',
        supply: '1',
        amount: '100',
        price: 2,
        creator: '0x000',
        tx_hash: '0x0000',
        is_deleted: 0,
        sold: 0,
    },
];