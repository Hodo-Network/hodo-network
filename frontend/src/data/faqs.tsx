import { FAQ } from '../typings/faq';

export const faqs: Array<FAQ> = [
  {
    id: 1,
    question: 'What is an NFT?',
    answer:
      'A non-fungible token (NFT) is a special type of cryptographic token which represents something unique; non-fungible tokens are thus not mutually interchangeable. This is in contrast to cryptocurrencies like bitcoin, and many network or utility tokens that are fungible in nature.',
  },
  {
    id: 2,
    question: 'What is a gNFT?',
    answer:
      'A georeferenced non-fungible token (gNFT) is a specialized NFT which references a real world location; georeferenced non-fungible tokens are thus not mutually interchangeable and are tied to the geographic location in which they are purchased and this location is tracked in the transaction. gNFTs can be referred to as digital souvenirs.',
  },
  {
    id: 3,
    question: 'How does the Hodo Network work?',
    answer:
      'The Hodo Network is a platform where you can buy and sell gNFTs. It is the first of it’s kind and utilizes the Avalanche network to track and manage the digital assets.',
  },
  {
    id: 4,
    question: 'How can I buy a Hodo Network gNFT?',
    answer:
      "First you need to create a MetaMask wallet, if you don’t already have one, and connect it to the Avalanche network (<a href='https://www.youtube.com/watch?v=uHrBWHNl2Nc'>https://www.youtube.com/watch?v=uHrBWHNl2Nc</a>). Once you have the MetaMask wallet configured you can tranfer AVAX from your AVAX web wallet to your MetaMask address on the C-Chain. You then need to link your MetaMask wallet to the Hodo Network via the connect button at Hodo Network. Once your MetaMask wallet is linked and funded with AVAX you can purchase gNFTs on the Hodo Network. If you need any help acquiring the AVAX token to purchase our gNFTs please reach out to our admin!",
  },
  {
    id: 5,
    question: 'Is AVAX the only token I can use to purchase a gNFT?',
    answer: 'Yes. This is the only supported token on the Hodo Network.',
  },
  {
    id: 6,
    question:
      'Can I buy a gNFT if I am not in the physical location the gNFT was minted for?',
    answer:
      'There is the ability of purchase gNFTs that have been minted specifically for a location other than the location you are currently in. If the user that is selling the gNFT allows a foreign buyer then you may purchase the gNFT. You should understand that a gNFT that is purchased from outside the area it was minted for will be devalued versus a gNFT that maintains the integrity of being purchased within the originally intended area (a pure gNFT). ',
  },
  {
    id: 7,
    question: 'What are the different classifications of a gNFT?',
    answer: 'There are 3 classifications for gNFTs: Epic, Rare and Common.',
  },
  {
    id: 8,
    question: 'What is a ‘pure’ gNFT?',
    answer:
      'A pure gNFT is a gNFT that has always been purchased by users that were, at the time of purchase, located in the geographical location that the gNFT was minted for. This is the most valuable gNFT. It proves that the owner has always been in the correct geographical location when purchasing.',
  },
  {
    id: 9,
    question: 'How long is each series available for?',
    answer:
      'Each series will be available directly from the Hodo Network for 12-36 months. After this time the tokens that have not been sold will be burnt, effectively limiting the supply of tokens. Each series will have a unique burn date.',
  },
  {
    id: 10,
    question: 'How often are new series released?',
    answer:
      'We are on target to release a new series every 2-4 months, each series will have a different specific geographical specification. Keep an eye on our social media feeds for release information on future series!',
  },
  {
    id: 11,
    question: 'How many of each type are minted?',
    answer: 'Quantities are different for each classification and series. ',
  },
  {
    id: 12,
    question: 'Can I transfer my gNFT to another platform?',
    answer:
      'Yes, Hodo gNFTs will be able to be transferred but know that if you sell the gNFT on another platform and it is not properly georeferenced it’s value will be reduced. We recommend selling all your gNFTs on the Hodo Network to maintain the integrity of the gNFT.',
  },
  {
    id: 13,
    question: 'What features do you have planned for the future?',
    answer: "Please visit the <a href='/roadmap'>roadmap</a>!",
  },
  {
    id: 14,
    question: 'Which blockchain does the marketplace use?',
    answer:
      'The marketplace was built using the Avalanche blockchain. Assets are minted on the C-Chain.',
  },
  {
    id: 15,
    question:
      'Why were the assets minted on the C-Chain instead of the X-Chain?',
    answer:
      'The C-Chain is EVM compatible which will allow interoperability with other EVM chains in the future. Using the C-Chain also allows us to enable wallets like MetaMask for a better user experience.',
  },
  {
    id: 16,
    question: 'What token standard do the NFTs use?',
    answer:
      'The NFTs were minted using the ERC1155 token standard. This allows for multiple tokens to be represented by a single smart contract and unlocks some cost saving features like sending multiple tokens in a single transaction.',
  },
];
