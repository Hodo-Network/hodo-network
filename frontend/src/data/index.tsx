import { collectibles } from "./collectibles";
import { collections } from "./collections";
import { user } from "./user";

export { collectibles, collections, user };
