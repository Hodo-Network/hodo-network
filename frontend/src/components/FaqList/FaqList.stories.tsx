import { Meta, Story } from '@storybook/react';

import { faqs } from '../../data/faqs';
import { FaqList, FaqListProps } from './FaqList';

export default {
  title: 'Components/FaqList',
  component: FaqList,
} as Meta;

const Template: Story<FaqListProps> = (args) => <FaqList {...args} />;

export const Default = Template.bind({});
Default.args = {
  items: faqs,
};
