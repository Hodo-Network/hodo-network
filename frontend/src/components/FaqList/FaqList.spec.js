import ReactDOM from "react-dom";
import { cleanup } from "@testing-library/react";
import renderer from "react-test-renderer";
import FaqList from "./index";
import { faqs } from "../../data/faqs";

afterEach(cleanup);

test("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<FaqList items={faqs} />, div);
});

test("matches snapshot", () => {
  const tree = renderer.create(<FaqList items={faqs} />).toJSON();
  expect(tree).toMatchSnapshot();
});
