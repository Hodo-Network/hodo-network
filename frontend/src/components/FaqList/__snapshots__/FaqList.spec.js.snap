// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<dl
  className="space-y-8 divide-y divide-gray-200 dark:divide-gray-800"
>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      What is an NFT?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        A non-fungible token (NFT) is a special type of cryptographic token which represents something unique; non-fungible tokens are thus not mutually interchangeable. This is in contrast to cryptocurrencies like bitcoin, and many network or utility tokens that are fungible in nature.
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      What is a gNFT?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        A georeferenced non-fungible token (gNFT) is a specialized NFT which references a real world location; georeferenced non-fungible tokens are thus not mutually interchangeable and are tied to the geographic location in which they are purchased and this location is tracked in the transaction. gNFTs can be referred to as digital souvenirs.
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      How does the Hodo Network work?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        The Hodo Network is a platform where you can buy and sell gNFTs. It is the first of it’s kind and utilizes the Avalanche network to track and manage the digital assets.
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      How can I buy a Hodo Network gNFT?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        First you need to create a MetaMask wallet, if you don’t already have one, and connect it to the Avalanche network (
        <a
          href="https://www.youtube.com/watch?v=uHrBWHNl2Nc"
        >
          https://www.youtube.com/watch?v=uHrBWHNl2Nc
        </a>
        ). Once you have the MetaMask wallet configured you can tranfer AVAX from your AVAX web wallet to your MetaMask address on the C-Chain. You then need to link your MetaMask wallet to the Hodo Network via the connect button at Hodo Network. Once your MetaMask wallet is linked and funded with AVAX you can purchase gNFTs on the Hodo Network. If you need any help acquiring the AVAX token to purchase our gNFTs please reach out to our admin!
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      Is AVAX the only token I can use to purchase a gNFT?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        Yes. This is the only supported token on the Hodo Network.
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      Can I buy a gNFT if I am not in the physical location the gNFT was minted for?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        There is the ability of purchase gNFTs that have been minted specifically for a location other than the location you are currently in. If the user that is selling the gNFT allows a foreign buyer then you may purchase the gNFT. You should understand that a gNFT that is purchased from outside the area it was minted for will be devalued versus a gNFT that maintains the integrity of being purchased within the originally intended area (a pure gNFT). 
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      What are the different classifications of a gNFT?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        There are 3 classifications for gNFTs: Epic, Rare and Common.
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      What is a ‘pure’ gNFT?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        A pure gNFT is a gNFT that has always been purchased by users that were, at the time of purchase, located in the geographical location that the gNFT was minted for. This is the most valuable gNFT. It proves that the owner has always been in the correct geographical location when purchasing.
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      How long is each series available for?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        Each series will be available directly from the Hodo Network for 12-36 months. After this time the tokens that have not been sold will be burnt, effectively limiting the supply of tokens. Each series will have a unique burn date.
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      How often are new series released?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        We are on target to release a new series every 2-4 months, each series will have a different specific geographical specification. Keep an eye on our social media feeds for release information on future series!
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      How many of each type are minted?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        Quantities are different for each classification and series. 
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      Can I transfer my gNFT to another platform?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        Yes, Hodo gNFTs will be able to be transferred but know that if you sell the gNFT on another platform and it is not properly georeferenced it’s value will be reduced. We recommend selling all your gNFTs on the Hodo Network to maintain the integrity of the gNFT.
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      What features do you have planned for the future?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        Please visit the 
        <a
          href="/roadmap"
        >
          roadmap
        </a>
        !
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      Which blockchain does the marketplace use?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        The marketplace was built using the Avalanche blockchain. Assets are minted on the C-Chain.
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      Why were the assets minted on the C-Chain instead of the X-Chain?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        The C-Chain is EVM compatible which will allow interoperability with other EVM chains in the future. Using the C-Chain also allows us to enable wallets like MetaMask for a better user experience.
      </p>
    </dd>
  </div>
  <div
    className="pt-6 md:grid md:grid-cols-12 md:gap-8"
  >
    <dt
      className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
    >
      What token standard do the NFTs use?
    </dt>
    <dd
      className="mt-2 md:mt-0 md:col-span-7 prose-blue"
    >
      <p
        className="text-base text-gray-700 dark:text-gray-200"
      >
        The NFTs were minted using the ERC1155 token standard. This allows for multiple tokens to be represented by a single smart contract and unlocks some cost saving features like sending multiple tokens in a single transaction.
      </p>
    </dd>
  </div>
</dl>
`;
